install-libsocket++.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
So the script is needed for the automated build system (via Travis) to be able to build a new dependency listed in the `rosinstall` script.

I'll make this script "private" by prefixing it with a dot, much like the existing `.travis.yml` file.